code/modules/culture_descriptor/culture/citizenship_hidden.dm

```diff
@@ -1,20 +1,20 @@
-/decl/cultural_info/culture/hidden
+/decl/cultural_info/citizenship/hidden
```

Comment:
Hard no on repathing this stuff. This will be a nightmare for pulling in changes from upstream.

Adding a new citizenship class as a many-to-many mapping of specific locations and cultures is a good idea - we should put our new cultures in mods/, not modify the base code.

Reply:
Chatted on discord,
> Part of repathing was something I talked to loaf about where the path "cultural_info/culture" is used in a couple of ways (explained on the pr but basically as a generic term And a category, so repathing the category has very low impact and makes things far more readable as it differentiates it from the bulk of "info/culture" which is the underlying system, and is probably something they'll be doing upstream also)